### DISCUSSION TITLE
Faner/Tabs

### DISCUSSION THREAD

**Discussion by onordbo** [fremtind/jokul#2342] [Annet]
Som et ledd i [inventory prosjektet](https://github.com/fremtind/jokul/projects/6) vårt ønsker vi tilbakemelding og stemmer fra både design og teknologi for å kartlegge hvilke komponenter og behov vi burde prioritere. Nedenfor følger en kort hypotese/slutning fra Designsystem-teamet, i tillegg til en rekke kommentarer vi har merket oss gjennom arbeidet.

Se på skjermbildene fra inventory [tilgjengelig her for referanse](https://www.figma.com/file/66kOJ5nCHjgtkfLW2sQWB8/Jøkul-Inventory-Innsiktsyntese?node-id=0%3A1). Legg gjerne til ønsker, behov, tanker, meninger og annet som kommenaterer i denne tråden, så vi kan sørge for at oppgavene blir så godt beriket med informasjon som mulig, dette gjør det enklere for alle involverte å ta tak i og løse de utfordringene vi møter sammen.

***

### TL;DR 👉 Generelt lite problemskapende komponent, brukes hovedsaklig av SAM, Flyt og litt Motor.

- Flere nivåer av faner - toppmeny og innhold
- Flyt (behov): Tabs i ulike nivåer
- SAM: Trengs faner blant annet til Felles tilbudsflyt
- Mulighet til å gjøre API-kall ved visning av en fane

**Comment by wkillerud** [comment]
Det er gjort noe arbeid på dette i #1958
